sdks/java/expansion-service/build.gradle

```diff
@@ -48,6 +48,11 @@ dependencies {
   implementation library.java.vendored_grpc_1_54_0
   implementation library.java.vendored_guava_32_1_2_jre
   implementation library.java.slf4j_api
+  // Stage default drivers for JdbcIO schema transforms
```

Comment:
This is the base class that everyone using an expansion service depends on (even if they have nothing to do with jdbc or even IOs at all). Let's add the dependency to the specific sdks:java:extensions:schemaio-expansion-service.

Reply:
That makes sense, moved.